test/subscribe.test.js

```diff
@@ -0,0 +1,89 @@
+describe('Subscribe', function() {
+  describe('PersistedModel', function() {
+    describe('configured to source changes locally', function() {
+      before(function() {
+        var test = this;
+        var app = loopback({localRegistry: true});
+        var ds = app.dataSource('ds', {connector: 'memory'});
+        this.Score = app.model('Score', {
+          dataSource: 'ds',
+          changeDataSource: false // use only local observers
+        });
+      });
+
+      it('should detect create', function(done) {
+        var Score = this.Score;
+
+        Score.subscribe(function(err, changes) {
+          changes.on('data', function(change) {
+            expect(change.type).to.equal('create');
+            changes.destroy();
```

Comment:
`changes.destroy` is not called in case the assertion fails (throws an error). I think you should call the destroy method first, before making any assertions. Please fix this in the other unit-tests too.